src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,10 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+class LetterRemover private constructor() {
+    companion object {
+        @JvmStatic
+        fun removeLetter(input: String): CharArray {
+            return input.lowercase().replace('r', '!').toCharArray()
```

Comment:
There some difference between expected and actual results.
In the task 
//result example
['!', 'e', 'm', 'o', 'v', 'e', ' ', 'l', 'e', 't', 't', 'e', '! ', 's']
Actual result 
!emove lette!s

How do you think why?